odl_video/envs_test.py

```diff
@@ -148,3 +150,13 @@ def test_get_key():
             b'/q0txR0v1rqmowS1mQIDAQAB\n'
             b'-----END PUBLIC KEY-----\n'
         )
+
+
+def test_parse_env():
+    testpath = 'testenv.txt'
```

Comment:
Can you use a `TemporaryFile` here or put it into a temporary directory? If it's not done with a context manager or something we should also put the `os.remove` in a `finally` block